launch_testing/launch_testing/tools/output.py

```diff
@@ -77,3 +82,49 @@ def expected_output_from_file(path):
             return [re.compile(regex) for regex in f.read().splitlines()]
 
     raise RuntimeError('could not find output check file: {}'.format(path))
+
+
+def expect_output(
+    text=None,
+    *,
+    lines=None,
+    expected_text=None,
+    expected_lines=None,
+    strip_ansi_escape_sequences=True,
+    strict=False
+):
+    """
+    Match output text or lines with expected text or lines.
+
+    Either (expected) text or (expected) lines can be provided but
+    giving both results in a ValueError.
+
+    :param expected_text: output text expectation, as supported
+        by `launch_testing.tools.text.build_text_match`
+    :param expected_lines: output lines expectation, as supported
+        by `launch_testing.tools.text.build_line_match`
+    :param text: output text to be matched
+    :param lines: output text lines to be matched
+    :param strip_ansi_escape_sequences: If True (default), strip
+        ansi escape sequences from actual output before comparing
+    """
+    if (text is not None) == (lines is not None):
+        raise ValueError('Either lines or text, but not both, must be specified')
+
+    if (expected_text is not None) == (expected_lines is not None):
+        raise ValueError('Either expected lines or text, but not both, must be specified')
+
+    if expected_text is not None:
+        if text is None:
+            text = '\n'.join(lines)
```

Comment:
IMO, documentation is not clear enough about this.
I thought you could use both `lines` and `expected_lines`, or `text` and `expected_text`. But I didn't get than combinations like `lines` and `expected_text` were valid.

Reply:
Done in d3982e54e9449d63d353bf0a00753b3489b75e7d